### DISCUSSION TITLE
Sign Up Fields

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#115] [3 - Design]
<img width="722" alt="{E2A8AAC7-087F-4EFD-B688-9C59D04F1337}" src="https://github.com/user-attachments/assets/b628a14b-82f9-43b0-a26c-6e1c9d7d0648">

@aripuguh 
We should add 
- "Your First Name" (Emmy as the text field pre-fill - or any other name really) * required
- "How did you hear about us?" (Text field)
- "Upload a profile picture - you can also do this later"
- Team Invite Code (ideally this is pre-filled using URL parameters in invite links @alice-ang and @Cristian-Nass know what this means @aripuguh  (I hope... I mean something like app.projectgeco.com/?abc-123)
  - "You can always join a team later. Or start your own team." <- description text for this input field.

to the sign up page.

We should also think about the "profile" and allowing people to add a profile picture there later.

**Comment by aripuguh** [comment]
How about this? 

<img width="444" alt="Screenshot 2024-10-07 at 10 30 27 AM" src="https://github.com/user-attachments/assets/5d9ddb30-c893-4ab8-b418-b197f5271423"> <img width="458" alt="Screenshot 2024-10-07 at 10 33 55 AM" src="https://github.com/user-attachments/assets/8d41d2ff-49d3-4e80-947e-ecff2883865f">


To make it less cluttered, I think the link part can be located after the signed-up process. What do you think? Or is it fundamental for them to know/write everything on one page?
